client/app/components/_mapPicker/mapPicker.component.js

```diff
@@ -0,0 +1,14 @@
+import template from "./mapPicker.html";
+import controller from "./mapPicker.controller";
+import "./mapPicker.styl";
+
+let mapPickerComponent = {
+  bindings: {
+    placeholder: "@",
+    data: "<" // item with latitude, longitude and address
```

Comment:
Ah, yeah, forgot to change it. I tried to implement something like:
```
data: "<",
getData: "&"
```
where getData is called from the parent scope and evaluated in the child scope - the "&" operator does exactly the opposite though (calling from child, evaluate in parent scope).

Therefore I just stuck to two way binding like _markdownInput is also using. Just forgot to change "<" to "=" .